### ISSUE TITLE
Schedules should be stored slightly differently

### ISSUE THREAD

**Issue by longhotsummer** [laws-africa/indigo#498]
The `doc` element should probably use the name `schedule` consistently and have another way of generating an ID.

* [x] migration to update old data
* [x] TOC builder must be updated
* [x] render correctly
* [x] fragments and quick edit

**Comment by longhotsummer** [comment]
Suggestion is that we move these into the `<act>` element as `<component>`s, rather than having a whole separate FRBR URI structure. Then, the id would be inherited from the component.

**Comment by longhotsummer** [comment]
Example:


eg.

```xml
<attachments>
  <attachment eId="att_1">
    <heading>Schedule</heading>
    <doc name="schedule">
      <meta>
        <identification source="#slaw">
          <FRBRWork>
            <FRBRthis value="/za/act/1980/01/!schedule"/>
            <FRBRuri value="/za/act/1980/01"/>
            <FRBRalias value="Schedule"/>
            <FRBRdate date="1980-01-01" name="Generation"/>
            <FRBRauthor href="#council"/>
            <FRBRcountry value="za"/>
          </FRBRWork>
          <FRBRExpression>
            <FRBRthis value="/za/act/1980/01/eng@/!schedule"/>
            <FRBRuri value="/za/act/1980/01/eng@"/>
            <FRBRdate date="1980-01-01" name="Generation"/>
            <FRBRauthor href="#council"/>
            <FRBRlanguage language="eng"/>
          </FRBRExpression>
          <FRBRManifestation>
            <FRBRthis value="/za/act/1980/01/eng@/!schedule"/>
            <FRBRuri value="/za/act/1980/01/eng@"/>
            <FRBRdate date="2020-05-12" name="Generation"/>
            <FRBRauthor href="#slaw"/>
          </FRBRManifestation>
        </identification>
      </meta>
      <mainBody>
        <section eId="sec_1">
        ...
        </section>
      </mainBody>
    </doc>
  </attachment>
</attachments>
```

* attachment
* heading goes inside attachment, removing need for hcontainer immediately inside the doc
* correct attachment id
* correct doc name and FRBR component

**Comment by longhotsummer** [comment]
We must migrate all schedules with this, even the really old ones. Then support for old schedules can removed completely.

**Comment by longhotsummer** [comment]
Fixed by #1079

**Comment by longhotsummer** [comment]
TOC builder updated by https://github.com/laws-africa/indigo/pull/1083
